### DISCUSSION TITLE
[Vote ended on 2022-06-07] Vote on https://github.com/ansible-community/community-topics/issues/89 ()

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#107] [Votes done]
Vote on merging https://github.com/ansible-community/antsibull/pull/429 and using it to build Ansible 6.0.0rc1
Related topic https://github.com/ansible-community/community-topics/issues/89

Please vote with +1/-1 until June 7, 2022.

**Comment by mariolenz** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by felixfontein** [comment]
I updated the description to say "...using it to build Ansible 6.0.0rc1", as I noticed that [there is no 6.0.0b3 release](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/roadmap/COLLECTIONS_6.rst#release-schedule): the release next week is 6.0.0rc1.

**Comment by cybette** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by Ompragash** [comment]
+1

**Comment by felixfontein** [comment]
The proposal has been accepted, thanks everyone!
